src/app/main.cpp

```diff
@@ -167,7 +167,7 @@ int main( int argc, char **argv )
   }
 
 #if WITH_SENTRY
-  sentry_wrapper::install_message_handler();
+  //sentry_wrapper::install_message_handler();
```

Comment:
What happened to that?

Reply:
@m-kuhn , I wanted to test whether the crash came from this specific part of the code. Crash still present so that answered the question :)

Reply:
```suggestion
  sentry_wrapper::install_message_handler();
```